Modules/Sources/WordPressUI/Views/Users/Views/UserDetailsView.swift

```diff
@@ -175,12 +193,124 @@ struct UserDetailsView: View {
             value: "Update",
             comment: "The 'Update' button to set a new password on the user profile"
         )
+
+        static let deleteUserConfirmationTitle = NSLocalizedString(
+            "userdetail.alert.deleteUserConfirmationTitle",
```

Comment:
(nit) `userDetails.*`

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23791/commits/1175235d20019cc3b2cb5dd50966c40a0f3c03c0